pqarrow/arrowutils/sort_test.go

```diff
@@ -11,73 +11,356 @@ import (
 )
 
 func TestSortRecord(t *testing.T) {
```

Comment:
I think it would also be great if we could add some benchmarks for these.

Reply:
I'm pretty new to `frostdb`, this probably my second or third week on the code base. While I'm comfortatable writing tests, I think I'm not qualified to write benchmarks yet.

I hope someone more knowledgable and better qualified should help with benchmarks. What I'm saying benchmarking is beyond the scope of this PR.

Reply:
How about now? 😂 I feel you are more than qualified given your recent contributions, but up to you.

Reply:
haha! lol! I think I can do it, will make sure we have a benchmark in place.